packages/main/src/util.ts

```diff
@@ -19,9 +19,15 @@
 import { BrowserWindow } from 'electron';
 import * as os from 'os';
 
-export const isWindows = os.platform() === 'win32';
-export const isMac = os.platform() === 'darwin';
-export const isLinux = os.platform() === 'linux';
+export function isWindows(): boolean {
```

Comment:
I'm wondering if we could have constants (not exported) so it's initialized only once

like

```ts
const windows = os.platform() === 'win32';
export function isWindows(): boolean {
    return windows;
}
```

Reply:
Sounds great!